handbook/company/senior-software-backend-engineer.md

```diff
@@ -0,0 +1,71 @@
+
+
+### Your primary responsibilities 🔍
+
+As a Senior Software Engineer at Fleet, you will get the chance to…
+
+In your first 90 days:
+
+- 🧑‍🔬 Design, Develop, Test and maintain a state-of-the-art Backend code including robust APIs to support mobile and desktop clients.
+- 🛠️ Write code and tests, build prototypes, resolve issues, and profile and analyze bottlenecks.
+- 💭 Manage and optimize scalable distributed systems in the cloud
+- 🤝 Collaborate closely with Product managers to understand requirements and translate them into work plans and software design.
+- 🚀 Actively participating in all engineering scrum meetings, including sprint planning, daily standups, sprint demos, sprint retrospectives, and estimation sessions.
+- 🌟 Contribute to the overall success of the [customer cxperience (CX)](https://fleetdm.com/handbook/company/development-groups#customer-experience-group) product group by ensuring users receive valuable new features.
+
+### Are you our new team member? 🧑‍🚀
+
+If most of these qualities sound like you, we would love to chat and see if we're a good fit.
+
+### You "get it":
+
+- 🦉 Translate requirements into proper design and software.
+- 🤝 Communicate regularly with stakeholders, project managers, quality assurance teams, and other developers regarding progress on long-term technology roadmap.
+- 🧪 Develop and oversee testing schedules in client server environment.
+- 🏃‍♂️ Familiarity with agile development processes and scrum methodologies.
+- 🛠️ Produce quality code, raising the bar for team performance and speed.
+- 📖 Mentor Junior team members.
+
+### You can "walk the walk":
+
+- 🤝 Collaboration: You work best in a participatory, team-based environment.
+- 📈 Produce quality code, raising the bar for team performance and speed
+- 🚀 Prototype-first: You embrace speed and failure as we iterate towards the right solution. You have hands-on experience in creating low and high fidelity prototypes. You’re comfortable accepting suboptimal designs in favor of iteration.
+- 🧬 Simplicity: You love complex questions and use your work to simplify that complexity for users.
+- 🛠️ Technical: You understand the software development processes. You understand that software quality matters.
+- 🟣 Openness: You are flexible and open to new ideas and ways of working.
+- ➕ Bonus: Cybersecurity or IT background.
+
+### You can "talk the talk":
+
+- 📖 Bachelor’s degree (or equivalent) in computer science
+- 💭 3-5 years' of experience in backend/SaaS development.
+- 🦉 You are Proficient in software backend development. You practice OOP design, design patterns and SOLID principles.
```

Comment:
I don't mind listing SOLID principles, but we should include something about them in the handbook first if we require it. We talk about agile, scrum, and lean software development in the handbook, so I'm suggesting that instead.

Reply:
SGTM 👍